test/ruby/test_zjit.rb

```diff
@@ -489,6 +489,21 @@ def test = [1,2,3]
     }
   end
 
+  def test_array_splat
+    assert_compiles '"foo"', %q{
+      def foo(b) = b
+      def test(a) = foo(*a)
```

Comment:
can you also test with splatting nil, calling custom `to_a`, ...?

Reply:
Added 👍